_dashboards/discover/multi-data-sources.md

```diff
@@ -33,60 +33,69 @@ To enable the feature:
+1. In the Dashboards console, choose **Index Patterns** > **Create index pattern**, as shown in the following image.
+   
+    <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-5.png" width=500 alt="Index pattern user interface">
+
+1. Choose **Use external data source connection**.
+1. Start typing in the **Search data sources** field to search for the data source you created earlier and then select the data source and **Next step**.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-6.png" width=500 alt="Index pattern search user interface">
+
+1. Add an **Index pattern name** to define the index pattern and then choose **Next step**.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-7.png" width=500 alt="Index pattern define user interface">
+
+1. Select an option for the **Time field** and then choose **Create index pattern**.
 
-2. Select **Use external data source connection**.
-3. Start typing in the Search data sources field to search for the data source you created earlier and then select the data source and **Next step**.
-<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-6.png" width=500 alt="Index pattern search user interface">
+    <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-8.png" width=500 alt="Index pattern time field user interface">
 
-4. Add an **Index pattern name** to define the index pattern and then select **Next step**.
-<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-7.png" width=500 alt="Index pattern define user interface">
+## Searching data
 
-5. Select an option for the **Time field** and then choose **Create index pattern**.
-<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-8.png" width=500 alt="Index pattern time field user interface">
+Before you start searching for data, set up the time filter. The sample index pattern used for this tutorial contains time-based data. You can set a time filter that displays only the data within a specified time range, and you can choose the time filter to change the time range or select a specific time range in the histogram.
 
-## Search data
+### Using the time filter
 
-Before you start searching for data, set up the time filter. The sample index pattern used for this tutorial contains time-based data. You can set a time filter that displays only the data within a specified time range, and you can select the time filter to change the time range or select a specific time range in the histogram.
+1. In the Dashboards console, choose **Discover** and confirm the index pattern being used is `opensearch_dashboards_sample_data_ecommerce`.
+1. Choose the calendar icon to change the time field. The default is **Last 15 minutes**.
+1. Change the time field to **Last 7 days** and choose **Refresh**.
 
-### Use the time filter
+    <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-9.png" alt="Time filter user interface">
```

Comment:
It looks like we need an "as shown in the following image" above this.

Reply:
Change made